### PR TITLE
Bug 1828668: Filtered out the noobaa based provisioner from storage class list

### PR DESCRIPTION
**Pull request by gnehapk**
Filtered out the noobaa based provisioner too from storage class list on ocs install view

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/src/components/modals/storage-class-dropdown.tsx [hunk 1]
```diff
@@ -5,7 +5,12 @@ import { StorageClassResourceKind } from '@console/internal/module/k8s';
 import { StorageClassDropdownInner } from '@console/internal/components/utils/storage-class-dropdown';
 import './storage-class-dropdown.scss';
 
-const cephStorageProvisioners = ['ceph.rook.io/block', 'cephfs.csi.ceph.com', 'rbd.csi.ceph.com'];
+const cephAndNoobaStorageProvisioners = [
+  'ceph.rook.io/block',
+  'cephfs.csi.ceph.com',
+  'rbd.csi.ceph.com',
+  'nooba.io/obc',
```
Comment (cloudbehl) [inline frontend/packages/ceph-storage-plugin/src/components/modals/storage-class-dropdown.tsx:12]:
```suggestion
  'noobaa.io/obc',
```

File: frontend/packages/ceph-storage-plugin/src/components/modals/storage-class-dropdown.tsx [hunk 2]
```diff
@@ -5,7 +5,12 @@ import { StorageClassResourceKind } from '@console/internal/module/k8s';
 import { StorageClassDropdownInner } from '@console/internal/components/utils/storage-class-dropdown';
 import './storage-class-dropdown.scss';
 
-const cephStorageProvisioners = ['ceph.rook.io/block', 'cephfs.csi.ceph.com', 'rbd.csi.ceph.com'];
+const cephAndNoobaStorageProvisioners = [
```
Comment (cloudbehl) [inline frontend/packages/ceph-storage-plugin/src/components/modals/storage-class-dropdown.tsx:8]:
```suggestion
const cephAndNoobaaStorageProvisioners = [
```

File: frontend/packages/ceph-storage-plugin/src/components/modals/storage-class-dropdown.tsx [hunk 3]
```diff
@@ -14,7 +19,9 @@ const StorageClassDropdown = (props: any) => {
   const scData = _.get(scConfig.resources.StorageClass, 'data', []) as StorageClassResourceKind[];
 
   const filteredSCData = scData.filter((sc: StorageClassResourceKind) =>
-    cephStorageProvisioners.every((provisioner: string) => !sc?.provisioner?.includes(provisioner)),
+    cephAndNoobaStorageProvisioners.every(
```
Comment (cloudbehl) [inline frontend/packages/ceph-storage-plugin/src/components/modals/storage-class-dropdown.tsx:22]:
```suggestion
    cephAndNoobaaStorageProvisioners.every(
```

**Comment by cloudbehl** [comment]
can you fix commit message & description of PR too?

File: frontend/packages/ceph-storage-plugin/src/components/modals/storage-class-dropdown.tsx (hunk 2 above)
Comment (bipuladh) [reply frontend/packages/ceph-storage-plugin/src/components/modals/storage-class-dropdown.tsx]: Do we really need this long name? Maybe something like `ocsProvisioners`?

Comment (cloudbehl) [reply frontend/packages/ceph-storage-plugin/src/components/modals/storage-class-dropdown.tsx]: makes sense!

**Pull request merged**